.github/workflows/ui_tests.yml

```diff
@@ -0,0 +1,28 @@
+---
+name: UI tests
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  source-build:
+    name: Run Netdata Agent UI tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Git clone repository
+        uses: actions/checkout@v2
+        with:
+          submodules: recursive
+      - name: Install the agent
+        run: >-
+          sudo apt-get install zlib1g-dev uuid-dev libuv1-dev liblz4-dev libjudy-dev libssl-dev libelf-dev libmnl-dev
+          gcc make git autoconf autoconf-archive autogen automake pkg-config curl python cmake &&
```

Comment:
We almost certainly want to be running `packaging/installer/install-required-packages.sh netdata --non-interactive--dont-wait` here instead of this, as that will ensure we properly track dependency changes.

Reply:
Done :)